lib/redis_memo/memoize_query.rb

```diff
@@ -9,10 +9,35 @@ module RedisMemo::MemoizeQuery
   require_relative 'memoize_query/invalidation'
   require_relative 'memoize_query/model_callback'
 
-  # Only editable columns will be used to create memos that are invalidatable
-  # after each record save
+  # Core entry method for using RedisMemo to cache SQL select queries on the given
+  # column names.
```

Comment:
Let's link the CachdSelect doc here, and briefly describe how it's done.

E.g. "we intercept any activerecord select queries and extract the dependencies from SQL query parameters. learn more in +CachedSelect+"